e2e/playwright/test-utils.ts

```diff
@@ -243,6 +243,18 @@ export async function getUtils(page: Page) {
         await closeDebugPanel(page)
       }
     },
+    getPixValue: async (x: number, y: number) => {
+      const buffer = await page.screenshot({
+        fullPage: true,
+      })
+      const screenshot = await PNG.sync.read(buffer)
+      const index = (screenshot.width * y + x) << 2
```

Comment:
Slick

Reply:
I stole this, I'm going to change it to `* 4` this is too cleaver imo.